src/main/java/io/jenkins/plugins/synopsys/security/scan/extension/pipeline/SecurityScanStep.java

```diff
@@ -219,7 +219,7 @@ public void setBlackduck_install_directory(String blackduck_install_directory) {
 
     @DataBoundSetter
     public void setBlackduck_scan_full(Boolean blackduck_scan_full) {
-        this.blackduck_scan_full = blackduck_scan_full ? true : null;
+        this.blackduck_scan_full = blackduck_scan_full ? true : false;
```

Comment:
If the parameter `blackduck_scan_full` is not even mention in the pipeline then it will run rapid scan because our plugin is setting it to `false` if `true` is not passed but it shouldn't be the case. If this parameter is not mentioned the Jenkinsfile then our plugin shouldn't do anything by default, because bridge will by default run full scan if nothing is passed. The previous bug was - for `false` value we were setting `null` value and for `null` value bridge runs full scan, so that's the problem.

Only removing the conditional part should work fine- 
`this.blackduck_scan_full = blackduck_scan_full`